rpc/grpc/server/src/connection.rs

```diff
@@ -121,11 +146,19 @@ impl Router {
             return Err(GrpcServerError::InvalidRequestPayload);
         }
         let rpc_op = request.payload.as_ref().unwrap().into();
+        let method = self.interface.get_method(&rpc_op);
```

Comment:
getting the method per request seems expensive to me. Maybe keep the policy in the routing map or do a static mapping from rpc_op to policy

Reply:
ok, let's store policies in the routing map